src/custom-element.js

```diff
@@ -1529,12 +1529,11 @@ function createBaseCustomElementClass(win) {
     /**
      * Turns the loading indicator on or off.
      * @param {boolean} state
-     * @param {{cleanup:boolean,force:boolean}=} opt_options
```

Comment:
I'll do the cleanup of making cleanup an argument to the function instead of a set of arguments in a follow up PR?

Reply:
Not relevant anymore